app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # associations
+  has_many :posts, foreign_key: 'author_id', inverse_of: 'author'
+  has_many :likes
+  has_many :comments
+
+  def most_recent_posts(num = 3)
+    posts.limit(num).order(created_at: :desc)
+  end
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, if you look closely, this function doesn't necessarily pick the `most_recent_posts`. It takes three `posts` and then, orders them. I suggest that you order all the posts first, and then pick the first three of that ordered list! 🤓 🔧 

 ```suggestion
  def most_recent_posts(num = 3)
    posts.order(created_at: :desc).limit(num)
  end
```